src/site/_includes/layout.njk

```diff
@@ -27,6 +27,7 @@ eleventyComputed:
     <link rel="preconnect" href="//firebaseinstallations.googleapis.com" crossorigin="" />
     <link rel="preconnect" href="//webdev.imgix.net" crossorigin="" />
 
+    <link rel="preload" as="script" crossorigin href="{{ preloadJS.path | stripQueryParamsDev }}">
```

Comment:
Unless the point where the script is used also has the crossorigin attribute, you probably don't want it here.